### PR TITLE
Ag 13219 mdoc code indenting

### PR DESCRIPTION
**Pull request by markdurrant**
Standardise on 4 spaces for indents in docs code blocks

### CODE REVIEW

File: documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc [hunk 1]
```diff
@@ -388,14 +397,18 @@ The example above demonstrates how to configure a single column. To apply this c
 ```jsx
 // Apply settings across all columns
 const defaultColDef = useMemo(() => ({
- filter: true // Enable filtering on all columns
+    filter: true // Enable filtering on all columns
 }))
 
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc]: Don't think this is intentional

File: documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc [hunk 2]
```diff
@@ -452,35 +465,39 @@ The example above demonstrates how to configure a single column. To apply this c
...
-     :defaultColDef="defaultColDef"
-   >
-   </ag-grid-vue>
-   `,
- // ...
+    template:
+        `
+        <ag-grid-vue
+            style="width: 100%; height: 100%"
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc [hunk 3]
```diff
@@ -526,11 +543,15 @@ So far we've covered creating a grid, updating the data within the grid, and con
 Grid Options are passed to the grid component directly as props. Let's enable pagination by adding `pagination={true}`:
 
 ```jsx
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc [hunk 4]
```diff
@@ -576,19 +597,23 @@ Grid Options are added directly to the `ag-grid-vue` component. Let's test this
...
-   >
-   </ag-grid-vue>
-   `,
- //...
+    name: "App",
+    template: 
+        `
+        <ag-grid-vue
+            style="width: 100%; height: 100%"
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc [hunk 5]
```diff
@@ -815,37 +840,43 @@ Let's try this by creating a new component to display the company logo in the 'c
...
- standalone: true,
- imports: [CommonModule],
- template: `<span>@if(value){<img [alt]="value" [src]="'https://www.ag-grid.com/example-assets/space-company-logos/' + value.toLowerCase() + '.png'" /> <p>{{ value }}</p>}</span>`,
- styles: ["img {display: block; width: 25px; height: auto; max-height: 50%; margin-right: 12px; filter: brightness(1.2);} span {display: flex; height: 100%; width: 100%; align-items: center} p { text-overflow: ellipsis; overflow: hidden; white-space: nowrap }"]
+    selector: 'app-company-logo-renderer',
+    standalone: true,
+    imports: [CommonModule],
+    template: `<span>@if(value){<img [alt]="value" [src]="'https://www.ag-grid.com/example-assets/space-company-logos/' + value.toLowerCase() + '.png'" /> <p>{{ value }}</p>}</span>`,
+    styles: ["img {display: block; width: 25px; height: auto; max-height: 50%; margin-right: 12px; filter: brightness(1.2);} span {display: flex; height: 100%; width: 100%; align-items: center} p { text-overflow: ellipsis; overflow: hidden; white-space: nowrap }"]
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc [hunk 6]
```diff
@@ -174,11 +174,20 @@ First we need to hook into the `gridReady` event by adding `(gridReady)="onGridR
 ```html
 <!-- The AG Grid component, with various Grid Option properties -->
 <ag-grid-angular
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/deep-dive/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 7]
```diff
@@ -81,14 +81,18 @@ Load the AG Grid library and create a container div. The div should have a heigh
...
-   <!-- Your Data Grid container -->
-   <div id="myGrid" style="height: 500px"></div>
- </body>
+    <head>
+        <!-- Includes all JS & CSS for the JavaScript Data Grid -->
+        <script src="https://cdn.jsdelivr.net/npm/ag-grid-community/dist/ag-grid-community.min.js"></script>
+    </head>
+    <body>
+        <!-- Your Data Grid container -->
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 8]
```diff
@@ -142,23 +146,27 @@ import { AgGridReact } from 'ag-grid-react'; // React Data Grid Component
...
+    ]);
 
+    // Column Definitions: Defines the columns to be displayed.
+    const [colDefs, setColDefs] = useState([
+        { field: "make" },
+        { field: "model" },
+        { field: "price" },
+        { field: "electric" }
+    ]);
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 9]
```diff
@@ -169,15 +177,26 @@ Rows and Columns are set as `AgGridReact` component attributes.
...
- <div
-  // define a height because the Data Grid will fill the size of the parent container
-  style={{ height: 500 }}
- >
-   <AgGridReact
-       rowData={rowData}
-       columnDefs={colDefs}
-   />
- </div>
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 10]
```diff
@@ -228,13 +247,36 @@ Rows and Columns are set as `ag-grid-angular` component attributes.
...
-   [rowData]="rowData"
-   [columnDefs]="colDefs" />
+    <!-- The AG Grid component -->
+    <ag-grid-angular
+        [rowData]="rowData"
+        [columnDefs]="colDefs" />
 `
 ```
 
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 11]
```diff
@@ -290,13 +332,17 @@ Rows and Columns are set as `ag-grid-vue` component attributes. Styling is appli
...
-   :columnDefs="colDefs"
-   style="height: 500px"
- >
- </ag-grid-vue>
+    <!-- The AG Grid component -->
+    <ag-grid-vue
+        :rowData="rowData"
+        :columnDefs="colDefs"
+        style="height: 500px"
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 12]
```diff
@@ -622,48 +668,94 @@ const myTheme = themeQuartz.withParams({
 {% if isFramework("react") %}
 ...
 return (
- <div>
-   <AgGridReact theme={myTheme} rowData={...} columnDefs={...} />
- </div>
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 13]
```diff
@@ -622,48 +668,94 @@ const myTheme = themeQuartz.withParams({
...
- template: `<div style="height: 100%; box-sizing: border-box;">
-   <ag-grid-angular
-     // ...
-     [theme]="theme"
-    />
- </div>`,
+    template: `<div style="height: 100%; box-sizing: border-box;">
+        <ag-grid-angular
+            // ...
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 14]
```diff
@@ -622,48 +668,94 @@ const myTheme = themeQuartz.withParams({
...
-               :theme="theme"></ag-grid-vue>
-           </div>
-       </div>
-   `,
+    template: `
+        <div style="height: 100%">
+            <div style="height: 100%; box-sizing: border-box;">
+                <ag-grid-vue
+                // ...
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 15]
```diff
@@ -622,48 +668,94 @@ const myTheme = themeQuartz.withParams({
...
+                :theme="theme"></ag-grid-vue>
+=======
+                :class="themeClass"></ag-grid-vue>
+>>>>>>> Stashed changes
+            </div>
+        </div>
+    `,
 // ...
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 16]
```diff
@@ -622,48 +668,94 @@ const myTheme = themeQuartz.withParams({
...
+    data: function () {
+        return {
+            // ...
+            themeClass: "ag-theme-quartz",
+        };
+    },
+>>>>>>> Stashed changes
 // ...
 {% /if %}
+<<<<<<< Updated upstream
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

File: documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc [hunk 17]
```diff
@@ -622,48 +668,94 @@ const myTheme = themeQuartz.withParams({
...
+    /* Changes the color of the grid text */
+    --ag-foreground-color: rgb(14, 68, 145);
+    /* Changes the color of the grid background */
+    --ag-background-color: rgb(241, 247, 255);
+    /* Changes the header color of the top row */
+    --ag-header-background-color: rgb(228, 237, 250);
+    /* Changes the hover color of the row*/
+    --ag-row-hover-color: rgb(216, 226, 255);
+}
+>>>>>>> Stashed changes
```
Comment (taktran) [inline documentation/ag-grid-docs/src/content/docs/getting-started/index.mdoc]: ditto

**Review by taktran — CHANGES_REQUESTED** [review]
Looks like there's some conflicts in here

**Comment by markdurrant** [comment]
> Looks like there's some conflicts in here

These should all be taken care of now.

**Review by taktran — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**